.github/workflows/build.yml

```diff
@@ -1,33 +1,37 @@
 name: Build & Test
 
-on: [push, pull_request]
+on: [push, pull_request, workflow_dispatch]
```

Comment:
With `workflow_dispatch`, we can trigger the action from a button in the GitHub UI for any branch.

Reply:
![image](https://user-images.githubusercontent.com/10584698/129853710-46ad2d45-2e5e-4521-a8d9-3426f2a7b86c.png)

Reply:
This is great!